src/export/ValueSetExporter.ts

```diff
@@ -57,7 +57,7 @@ export class ValueSetExporter {
         if (component.from.system) {
           const systemParts = component.from.system.split('|');
           const foundSystem = (
-            this.fisher.fishForMetadata(systemParts[0], Type.CodeSystem)?.url ??
+            this.fisher.fishForMetadata(component.from.system, Type.CodeSystem)?.url ??
```

Comment:
I see that this change makes it more consistent w/ the approach we take for value sets below -- so that's good.

I also see that if there is a version, we drop the version when there's a match but keep it when there isn't (value sets below does the same thing). I think this is one of those places where we want to keep the version when there is a match. I think we had decided, however, that these kinds of changes aren't necessarily in scope for this PR and can be deferred to another PR if we want.

Reply:
Now that I have your nifty regex replace in my back pocket, I don't mind doing it here. But I can also hold off for a separate PR if that's preferred.

Reply:
Actually, wait, now I'm not sure if I agree we should keep the version. The version the author specified gets added to the `composeElement.version` regardless of if we find a match on line 63. And because the `foundSystem` variable does a final `split('|')` and then only `foundSystem[0]` is used later, the version is always spliced off and never added to `composeElement.system`. So do we want to include the `|version` on the `system` property, as well as on the `version` property?

Reply:
Oh, dang. You're right! For whatever reason, `ValueSet.compose.include` splits the version off into its own element. And the `system` element is `uri` _not_ `canonical`. So, yep, we definitely do _not_ want to attach the `|version`.  Good call!